### ISSUE TITLE
can't install app: error code 1620 

### ISSUE THREAD

**Issue by longtth** [just-install/just-install#284]
I've tried to install apps using `just-install`, but got error 1620, 
Could you help me to have a look on it? 
```
C:\WINDOWS\system32>just-install tortoisesvn
2018/12/05 01:42:27 x86_64 - https://osdn.net/frs/redir.php?m=rwthaachen&f=%2Fstorage%2Fg%2Ft%2Fto%2Ftortoisesvn%2F1.11.0%2FApplication%2FTortoiseSVN-1.11.0.28416-x64-svn-1.11.0.msi
2018/12/05 01:42:27 Running msiexec.exe /q /i C:\Users\ASUS\AppData\Local\Temp\just-install\47681453.msi ALLUSERS=1 REBOOT=ReallySuppress
2018/12/05 01:42:27 exit status 1620
```

**Comment by lvillani** [comment]
For some reason the download gets truncated at around 85-90%

**Comment by lvillani** [comment]
SHA256 checksum are on the roadmap. The real issue here is that we are silently truncating downloads without reporting errors to the user. I think I found the piece of code responsible and I'm amazed it never raised an issue before :sweat_smile:

**Comment by lvillani** [comment]
After adding a bunch of error checks (shame of me for not adding them in the first place) I got this:

```
C:\Users\Lorenzo\go\src\github.com\just-install\just-install>.\just-install crystaldisk-info
2018/12/17 08:48:32 x86_64 - http://osdn.dl.osdn.jp/crystaldiskinfo/70295/CrystalDiskInfo8_0_0.exe
 2.24 MiB / 3.76 MiB [========================================>---------------------------]  59.63% 254.96 KiB/s 00m06s
2018/12/17 08:48:42 Error downloading file: net/http: request canceled (Client.Timeout exceeded while reading body)
```

So... we are setting a 10-second timeout in the `http.Client` instance we use ([see here](https://github.com/just-install/just-install/blob/121a74eddd9f7e197d12169e7ca6bd23c853394d/pkg/justinstall/util.go#L220-L223)) which ends up aborting the request after 10 seconds even when we are actively consuming data via `os.Copy`.

Said timeout has been in place for at least one year and I cannot recall ever seeing this problem. I'm not sure whether we always relied on a bug within Go's stdlib. It's clear to me now that we are setting timeouts the wrong way and that we should probably set timeouts at the `Transport` level.

**Comment by lvillani** [comment]
This should be fixed in v3.4.3
